migrations.py

```diff
@@ -93,3 +93,22 @@ async def m006_add_zeroconf_column(db):
         )
     except OperationalError:
         pass
+
+
+async def m007_add_pending_column(db):
```

Comment:
- if I understand correctly this change is unrelated to the this PR
   - it was supposed to be here: https://github.com/lnbits/satspay/pull/22/files#diff-d7af58842a76f9c7d30d94d9e2539190b7f4d85a238ef85dc70e4dad3b026884
- we should have a release with this change alone ASAP (since it is a blocker bug)